src/components/address-finder/index.njk

```diff
@@ -0,0 +1,86 @@
+---
+title: Address finder
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+
+The address finder component uses the ONS Address Index API to search for and return addresses. 
+
+## When to use this component
+
+Use the address finder when you need users to find and confirm a UK address. This could be their own or any other address that is required for the question.
+
+## When not to use this component
+
+Do not use this component if non-UK addresses are required for the question.
+
+## How to use this component
+
+The address finder extends the code base used for the [autosuggest component](/components/autosuggest). 
+
+There are two main ways to use this component. Addresses can either be editable or non-editable.
+
+### Editable addresses
+
+Include the parameter `isEditable: true`. Note that there are other required parameters that can be viewed on the Nunjucks output of the example.
+
+When an address has been selected, the “mode” of the component will change to show each line of the address as individual, manual inputs. This allows the user to check and change the address.
+
+If the address is amended, the UPRN (unique ID) will be removed and not submitted.
+
+{{
```

Comment:
I think it's worth stating that the manual inputs will be shown if there's an error with the API.  Until we sort out the JWT, users of this will never see the lookup in the example, so it just looks like the wrong example.

Reply:
We do in the validation section